docs/DeveloperGuide.adoc

```diff
@@ -798,8 +798,10 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 * Leow Wei Ching
 * Marcus Chen
+** Minor enhancement: Implemented `backup` command
```

Comment:
Might wanna consider removing this part and rebasing your branch with the upstream's master to resolve the merge conflict?